### PR TITLE
Added temperament (just, equal, pytha)

### PR DESCRIPTION
**Pull request by valentinradu**
Hi, 

I've added 3 temperaments that can be easily expanded to more if needed (one only needs the ratios).
I decided to put the temperament as global var (similar to concertA) since it seems to me it's not something related to any other context but the global context.

### CODE REVIEW

**Comment by valentinradu** [comment]
Also added a translation unit for the most common music notation systems. Chinese would be nice as well, but it's a mess. Maybe in the future. Cheers.

File: MusicKit/Pitch.swift [hunk 1]
```diff
@@ -28,28 +26,28 @@ public struct Pitch: Comparable {
...
-
-    var noteNameTuple: (LetterName, Accidental, Int)? {
-        return chroma.flatMap {
-            $0.names.first.map {
-                noteNameWithOctave(octaveNumber, nameTuple: $0) } }
+    
+    /// A value between -0.5 and 0.5 representing
+    /// the current deviation (e.g. for midi 60.5
+    /// deviation is 0.5)
+    public var deviation: Float {
```
Comment (0thernet) [inline MusicKit/Pitch.swift:40]: 👌

File: MusicKit/Temperament.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        var ratios = self.ratios
+        // We need to close the ratios (add 2) in order to
+        // find the last one (which should be always smaller than 2)
+        ratios.append(2.0)
+        var midi = Double.infinity
+        // This is the deviation from the chroma value. So, this
+        // function will output microtonal midi values (e.g 60.24)
+        // Probably there's a specific function that describes best
+        // the intermpolation for each temperament, but for now,
+        // we will simply use a linear interpolation
```
Comment (0thernet) [inline MusicKit/Temperament.swift:59]: 👍

**Review by 0thernet — APPROVED** [review]
This is awesome, thanks for the contribution! ✨

**Pull request merged**